README.md

```diff
@@ -51,9 +53,28 @@ const trades = await cowSdk.cowApi.getOrders({
   offset: 0,
 })
 console.log(trades)
-```
+````
+
+The SDK will expose:
+
+- The CoW API (`cowSdk.cowApi`)
+- The CoW Subgraph (`cowSdk.cowSubgraphApi`)
+- Convenient method to facilitate signing orders (i.e `cowSdk.signOrder`)
```

Comment:
This is no longer true in the latest version
Shouldn't it be updated as well?

Reply:
Yes, this PR reduce the documentation, I didn't actually fix the new names. I assume @shoom3301 will update once he does the vanilla JS example